unbound-generics.cabal

```diff
@@ -61,7 +61,7 @@ library
                        containers >= 0.5 && < 0.7,
                        contravariant >= 0.5,
                        profunctors >= 4.0,
-                       ansi-wl-pprint >= 0.6.7.2 && < 0.7,
+                       ansi-wl-pprint >= 1.0.2 && < 1.1,
```

Comment:
Do we really need to bump the lower bound? That might create problems for someone downstream of us

Reply:
Ah I had my Application dev :tophat: on and the issue said 
> bump our ansi-wl-pprint dependency to the latest (1.0.2)

So I figured that meant we didn't want earlier versions but yeh I see no issue with keeping a wider version range.